src/pages/index.js

```diff
@@ -29,47 +37,108 @@ export const cardSection = new Section(
 function fillProfileInputs() {
   const currentInfo = user.getUserInfo();
   nameInput.value = currentInfo.name;
-  titleInput.value = currentInfo.job;
+  titleInput.value = currentInfo.about;
+}
+
+function handleProfilePicSubmit(data) {
+  profilePicModal.setButtonText("Saving...");
+  api
+    .updateProfilePicture(data)
+    .then((res) => {
+      user.setUserInfo(res);
```

Comment:
You should update user's avatar here using the new method for updating avatar in UserInfo class